### ISSUE TITLE
Content-Type doesn't support charset

### ISSUE THREAD

**Issue by tommay** [kenichi/angelo#45]
The Content-Type header should always include a charset, at least for html, to prevent exploits.  Angelo needs a way to at least specify a charset, and it would be wise to provide a default charset where appropriate for vulnerable Content-Types like text/html.

**Comment by stugol** [comment]
+1

**Comment by kenichi** [comment]
No, it's not quite dead yet! 😉 More, it has been waiting on changes to upstream projects, mostly reel. If you have time to make a PR, it would be much appreciated. Thanks!
